flake.nix

```diff
@@ -0,0 +1,29 @@
+{
+  description = "Grammatical Framework's Resouce Grammar Library";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+  
+  inputs.gf-core.url ="github:BeFunctional/gf-core?ref=tp_nix_flake";
```

Comment:
This should probably not be merged until https://github.com/GrammaticalFramework/gf-core/pull/165 is merged and the urls are updated to use the official gf-core repo instead of a fork

Reply:
Sounds good.  Will update this mr then.